features/registry/registry.feature

```diff
@@ -173,3 +173,68 @@ Feature: Testing registry
       | Mirroring completed in |
     Given the "myimage" image stream was created
     And the "myimage" image stream becomes ready
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-29693
+  @admin
+  Scenario: [Disconnect]Import image from a secure registry using node credentials
+    Given I have a project
+    And evaluation of `image_content_source_policy('image-policy-aosqe').mirror_registry(cached: false, user: admin)` is stored in the :mirror_registry clipboard
```

Comment:
We do not need `user: admin` there.

Reply:
@liangxia Updated as comment, and it passed.